src/gluonts/core/component.py

```diff
@@ -506,6 +508,21 @@ def get_mxnet_context(gpu_number=0) -> mx.Context:
         return mx.context.cpu()
 
 
+def normalize_ctx(
+    ctx: Optional[Union[str, mx.Context, List[mx.Context]]]
```

Comment:
```suggestion
    ctx: Union[str, ContextType]
```

Reply:
The idea is to keep the `Optional` and `str` in `ContextType` to make it compatible with mxnet.

Reply:
Since `ContextType` already includes `Optional`, we can shorten this.